TestCases/hom_navierstokes/SphereViscous/nPoly3_QuadDominant/fem_Sphere.cfg

```diff
@@ -0,0 +1,188 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %

```

Comment:
Not all of them were supposed to get in there. We will have a look which ones to keep. Thanks for pointing it out.

Reply:
Hey, sorry, I'm a little new to the community. But is there a reason to cut down on the test cases? Starting out using SU2, I always found it useful to have a number of test cases, just to explore the breadth of the config options that are available. I also found it helpful in validating the code and it's performance.

Reply:
The main reason is to reduce the size of the repositories (mostly the separate repo for geometry and solution files) - we definitely want to provide examples of all features, and provide tests that will show the error in the regression tests if some feature is broken (that's what 'full coverage' meant) but since each GitHub repo has a max memory limit, and because we don't want to overload the user's system either, it's generally a good idea to try to cover multiple features in a single test case and/or use multiple config files that use the same mesh with different options, if that can be done without leaving out something useful. This is also why the test meshes are usually very coarse. The trade-off between memory/regression test time vs full coverage/providing examples is often debated.
*deleted/reposted to put it next to the actual question

Reply:
Agreed.. the general philosophy is that the continuous integration from Travis should just be a check for correctness (and maybe performance, although we don't do this yet here), so it is really about making tests that are as small as possible while still covering the features you want to protect. We want to keep the Travis turnaround time (and files sizes) to a minimum so as not to inhibit development.

This is not to be confused with proper V&V cases or educational cases/tutorials, which will require larger meshes running to convergence, best practice settings and methods, etc., etc.

If there are DG cases that are useful for the latter, we can turn them into tutorials or save them elsewhere so folks can still access. Or, just leave them in place but remove those tests from the regression scripts to save time on the tests, if they are redundant checks from a correctness standpoint.